client/scss/components/_gif_video.scss

```diff
@@ -2,6 +2,10 @@
   width: 100%;
 }
 
+.gif-video__caption p:last-child {
```

Comment:
Maybe  a `no-last-p-bottom-margin` (or better-named) root scope class make sense?

Reply:
Let's leave it like this for now until we use it in a few places?